### DISCUSSION TITLE
I am creating an offboard app for PX4 and need a camera image as input information for the app.

### DISCUSSION THREAD

**Discussion by takata1021** [microsoft/AirSim#4343] [Support Q&A]
I am creating an offboard app for PX4 and need a camera image as input information for the app.
I want to run the app on Airsim, but I don't know how to get the image.
Q1: Is it possible to get images from AirSim from MavSDK via Mavlink?

Q2: Can I get all the images that can be obtained from AirSim with MavLinkCom at https://microsoft-github-io.translate.goog/AirSim/image_apis/?_x_tr_sl=en&_x_tr_tl=ja&_x_tr_hl=ja&_x_tr_pto=sc? Please tell me the API to get.

**Comment by Bezet1** [comment]
Hi, did you get an anwer for this problem?
